org-cyf-itp/content/user-data/sprints/3/prep/index.md

```diff
@@ -5,6 +5,10 @@ emoji= '🧑🏾‍💻'
 menu_level = ['sprint']
 weight = 1
 [[blocks]]
+name="Record a Goose"
+src="https://github.com/CodeYourFuture/Project-Record-A-Goose/readme"
```

Comment:
I'd like to split this into two files - a README and a DEVELOPING - I don't think "how to run in docker" and "Feedback on the project is welcome" should be things that show up in the curriculum... WDYT?

Reply:
agreed, have done this now